src/olympia/devhub/forms.py

```diff
@@ -859,7 +859,7 @@ def get_context(self, name, value, attrs):
 class VersionForm(WithSourceMixin, forms.ModelForm):
     release_notes = TransField(widget=TransTextarea(), required=False)
     approval_notes = forms.CharField(
-        widget=TranslationTextarea(attrs={'rows': 4}), required=False
+        widget=forms.Textarea(attrs={'rows': 4}), required=False
```

Comment:
Drive-by because some test was initially failing because of approval notes so I was digging around there. Approval notes is just a `CharField` in the model, there shouldn't be any reason to use a `TranslationTextarea`.